jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/model/EnumValueDefinition.java

```diff
@@ -1,3 +1,19 @@
+/**
```

Comment:
I added this changes because everytime the build ran they came back, so I thought it would be nice for everyone to not have to exclude these again.